suse_migration_services/units/reboot.py

```diff
@@ -66,16 +66,13 @@ def main():
                         )
                     )
                 )
+            if not MigrationConfig().is_soft_reboot_requested():
+                command_list = ['systemctl', 'reboot']
+            else:
+                command_list = ['systemctl', 'kexec']
             log.info(
-                # reboot is performed through systemd. The call through
-                # systemd checks if there is a kexec loaded kernel and
-                # transparently turns 'systemctl reboot' into
-                # 'systemctl kexec'. Thus both ways, soft and hard
-                # reboot are managed in one call.
                 'Reboot system: {0}{1}'.format(
-                    os.linesep, Command.run(
-                        ['systemctl', 'reboot']
-                    )
+                    os.linesep, Command.run(command_list)
```

Comment:
the comment about could make this
```py
os.linesep, Command.run(
    ['systemctl', restart]
)
```
so no `command_list` variable and it could be more explicit

Reply:
Good point, I've adapted that part, so it is a nicer read